Shared/ca.lproj/Localizable.strings

```diff
@@ -921,9 +921,6 @@
 /* Toast displayed to the user after adding the item to their reading list. */
 "Menu.AddToReadingList.Confirm" = "S'ha afegit a la llista de lectura";
 
-/* Label for the button, displayed in the menu, takes you to sync sign in when pressed. */
-"Menu.BackUpAndSync.Label" = "Fes una còpia de seguretat i sincronitza les dades";
-
```

Comment:
Reminder: To make sure that we are okay here as some of these strings are completely removed instead of being replaced

Reply:
I confirm that the strings "Menu.BackUpAndSync.Label", "Menu.RemovePin.Confirm2" and "Menu.AddBookmarkAction2.Title" aren't used anymore in our codebase. Since the strings needed to be updated, I too the opportunity to align the name with the other strings name we already add.